test/AutoDiff/SIL/Serialization/sil_differentiability_witness.sil

```diff
@@ -161,7 +161,7 @@ sil_differentiability_witness hidden [parameters 0 1] [results 0] <τ_0_0 where
 // ROUNDTRIP:   vjp: @AD__generic__vjp_src_0_wrt_0_1 : $@convention(thin) <τ_0_0 where τ_0_0 : Differentiable> (@in_guaranteed τ_0_0, Float) -> (@out τ_0_0, @owned @callee_guaranteed (@in_guaranteed τ_0_0.TangentVector) -> (@out τ_0_0.TangentVector, Float))
 // ROUNDTRIP: }
 
-// IRGEN-LABEL: @AD__generic_PSSRS16_Differentiation14DifferentiableRzl = hidden global { i8*, i8* } {
+// IRGEN: @AD__generic_PSSRS{{16_Differentiation|s}}14DifferentiableRzl = hidden global { i8*, i8* } {
```

Comment:
Is there a reason why `-LABEL` was removed?

Reply:
I was running into a matching issue (I expect the regular expression was throwing it off).